controllers/test.go

```diff
@@ -3,10 +3,12 @@ package controllers
 import (
 	"encoding/json"
 	"fmt"
+	"strconv"
 
 	"github.com/astaxie/beego"
 
 	"github.com/microsoft/mouselog/detect"
+	jsonutils "github.com/microsoft/mouselog/file"
```

Comment:
Why rename to `jsonutils`?

Reply:
'file' is a very general identifier. Actually it is better to name the module "fileutils"

Reply:
OK.